src/blender_mcp/server.py

```
@@ -266,26 +290,69 @@ def get_object_info(ctx: Context, object_name: str) -> str:
+        # Create temp file path
+        temp_dir = tempfile.gettempdir()
+        temp_path = os.path.join(temp_dir, f"blender_screenshot_{os.getpid()}.png")
+        
+        result = blender.send_command("get_viewport_screenshot", {
+            "max_size": max_size,
+            "filepath": temp_path,
+            "format": "png"
+        })
+        
+        if "error" in result:
+            raise Exception(result["error"])
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)